configuration.go

```diff
@@ -75,16 +103,17 @@ type Configuration struct {
 	// Whether bugsnag should notify synchronously. This defaults to false which
 	// causes bugsnag-go to spawn a new goroutine for each notification.
 	Synchronous bool
+	// Whether the notifier should send all sessions recorded so far to Bugsnag
+	// when repanicking to ensure that no session information is lost in a
+	// fatal crash.
+	flushSessionsOnRepanic bool
```

Comment:
Booleans are really awkward configuration options in this notifier. Instead of addressing this issue (non-trivial, probably breaking changes required) or exposing another hard-to-understand configuration option, I've exposed a function that sets this property, and have it set to true by default - like you suggested. 

However, we don't always want to do this because this could lead to a very high amount of network traffic for services that have a lot of panics that are handled by a framework. We only want to flush in the case that we're pretty sure the application will die. Not when we just see a panic, because there might be a framework panic handler to catch it. We disable this setting in all of our framework integrations.